hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -225,26 +216,64 @@ protected Bytes computeAssociateTokens(final Bytes input, final MessageFrame mes
 		return null;
 	}
 
-	@SuppressWarnings("unused")
+	//	TODO: Abstract out the invocation of the ledgers as a helper method to avoid duplication
```

Comment:
```suggestion
	//	TODO: Abstract out the invocation of the ledgers as a helper method to avoid duplication
```
Intoduce two new methods:
```
	private AccountStore createAccountStore(WorldLedgers ledgers) {
		return new AccountStore(validator, dynamicProperties, ledgers.accounts());
	}

	private TypedTokenStore createTokenStore(WorldLedgers ledgers, AccountStore accountStore) {
		return new TypedTokenStore(accountStore, ledgers.tokens(), ledgers.nfts(), ledgers.tokenRels(),
				uniqTokenViewsManager, legacyStoreDelegate, delegate, new SideEffectsTracker());
	}
```